src/Umbraco.Tests.AcceptanceTest/cypress/integration/Tour/systemInformation.ts

```diff
@@ -0,0 +1,39 @@
+/// <reference types="Cypress" />
+
+function openSystemInformation(){
+    cy.get('[data-element="global-help"]').click();
+    cy.get('.umb-help-list-item').should('be.visible').click();
+    cy.get('.umb-drawer-content').scrollTo('bottom', {ensureScrollable : false});
+}
+
+context('System Information', () => {
+
+    beforeEach(() => {
+        //arrange
+        cy.umbracoLogin(Cypress.env('username'), Cypress.env('password'));
+    });
+
+    it('Check System Info Displays', () => {
+        openSystemInformation();
+        cy.get('.table').find('tr').should('have.length', 10)
+    });
+
+    it('Checks language displays correctly after switching', () => {
```

Comment:
Since this test is `'Checks language displays correctly after switching'` shouldn't we also test that it actually displays correctly before switching?